src/android/build-extras.gradle

```diff
@@ -0,0 +1,50 @@
+// Needed to speed up asset listing: refer to https://github.com/apache/cordova-plugin-file/blob/76c8fd5e43598ea660c2db64eb08e2659b45a575/src/android/AssetFilesystem.java#L41
```

Comment:
Do we have any numbers on how slow the asset listing is with/without this extra build step?